manifests/config.pp

```diff
@@ -36,6 +50,205 @@
       provider  => 'powershell',
     }
 
+    # Sync Notification Settings
+    exec { 'wsus-config-sync-notification-settings':
+      #Set sync to true or false
+      command   => "\$ErrorActionPreference = \"Stop\"
+                    \$wsusEmailNotificationConfiguration = (Get-WsusServer).GetEmailNotificationConfiguration()
+                    \$wsusEmailNotificationConfiguration.SendSyncNotification = \$${send_sync_notification}
+                    \$wsusEmailNotificationConfiguration.Save()
+                    return 'SendSyncNotification setting updated'
```

Comment:
Can we remove this?  Doubt it's needed.

Reply:
Remove which bit?

This sets the "Send email on sync update", i.e turning the feature on or off

Reply:
I'm referring to the `return 'SendSyncNotification setting updated'` piece since it is not necessary

Reply:
Oh OK - I've removed them 

I did quite like that it output that info when running `puppet agent -tv` manually